lib/action_policy/graphql/authorized_field.rb

```diff
@@ -23,14 +23,12 @@ def extract_option(key, &default)
 
       class AuthorizeExtension < Extension
         def apply
-          @to = extract_option(:to) { ::ActionPolicy::GraphQL.default_authorize_rule }
-          @raise = extract_option(:raise) do
-            if field.mutation
-              ::ActionPolicy::GraphQL.authorize_mutation_raise_exception
-            else
-              ::ActionPolicy::GraphQL.authorize_raise_exception
-            end
+          if field.mutation
```

Comment:
Let's prevent deprecations avalanche by showing this warning only once.
See, for example, https://github.com/ai/autoprefixer-rails/pull/173

Reply:
btw, would you prefer to use `ActiveSupport::Deprecation` here?

Reply:
Nope, we do not have ActiveSupport as a dependency.

We can add a conditional method to show a deprecation, though:

```ruby
if defined?(ActiveSupport::Deprecation)
  def show_deprecation
    ActiveSupport...
  end
else
  def show_deprecation
     warn ...
  end
end
```